utils/calcRecoveryRate.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Calculates the recovery rate and puts it in a nice looking string format.
+ * 
+ * @param  {int}     numDeaths     The number of deaths for a unit
+ * @param  {int}     numRecoveries The number of recoveries for a unit
+ * @return {String}  A parsed string percentage string of recoveries rounded to one decimal place.
+ */
+function calcRecoveryRate(numDeaths, numRecoveries) {
+    return ((numRecoveries / (numDeaths + numRecoveries)) * 100).toFixed(1) + '%';
+}
+
+/**
+ * Calculates the recovery rate from an array of data and splices the recovery rate
+ * into the middle of the array for table formatting.
+ * 
+ * @param  {Array} one data about the element we are processing. (e.g. state, country, etc.)
+ * @return {Array} A spliced array almost as it was before but now with a string formatted recovery rate.
+ */
+function spliceRecoveryRate(one) {
```

Comment:
`one` is a poor name, as a reader I have no idea what that means. 
Perhaps `area` would be better?

Reply:
`one` is the naming convention [already in use](https://github.com/ahmadawais/corona-cli/blob/75a4b514a32a223958c0ffa644103858e7ad9eb6/utils/getAll.js#L18) in this project elsewhere for a single entry.

Reply:
I believe we can make `one` better everywhere.